src/composites/FeaturesContainer/Overview/OverviewCollector.ts

```diff
@@ -0,0 +1,103 @@
+import { overviewData } from './index.ts';
+import { State } from "../../../state.ts";
+import { useAtomValue } from "jotai";
+
+//expected DSStats for compact-output: 0-0-0-0-6 / 1-1-0-0-34 / 4-0-1-0-161
+
+export function OverviewDataCollector():overviewData {
+    const dataset = useAtomValue(State.dataset);
```

Comment:
could we make `dataset` an argument to this function (`OverViewDataCollector: (dataset: schema.base.Dataset) => OverViewData`)? In general, hooks like `useAtomValue` cannot be used outside of react components.  This function will only work as expected if OverviewDataCollector is called in a component, but if we use it somewhere else, lets say in a unit test, it will break.

Reply:
also, could we rename `OverviewDataCollector` to something like `collectOverviewData`? Generally prefer to use `verbNoun` style for function names.

Reply:
Function renamed and data type defined for all overview elements using the dataset